listenbrainz/config.py.sample

```diff
@@ -19,6 +19,7 @@ PG_QUERY_TIMEOUT = "3000"
 # Set to True to enable 'synchronous_commit' for Postgres. Default: False
 PG_ASYNC_LISTEN_COMMIT = False
 
+MUSICBRAINZ_DB_URI = ""
```

Comment:
Does this need to be set in the ctmpl above?

Reply:
We have it already set in the consul template from when we were importing musicbrainz row IDs.

https://github.com/metabrainz/listenbrainz-server/blob/master/consul_config.py.ctmpl#L24

Reply:
kewl.